docs/admin/daemons.md

```diff
@@ -91,6 +88,14 @@ when the pod is created, so it is ignored by the scheduler).  Therefore:
    by the DaemonSet controller.
  - DaemonSet controller can make pods even when the scheduler has not been started, which can help cluster
    bootstrap.
+   
+Daemon pods do respect [taints and tolerations](/docs/user-guide/node-selection/index.md), but they are
+created with tolerations for the `node.alpha.kubernetes.io/notReady` and `node.alpha.kubernetes.io/unreachable`
```

Comment:
How about 

created with `NoExecute` tolerations for `node.alpha.kubernetes.io/unreachable` and `node.alpha.kubernetes.io/notReady` with no `tolerationSeconds`.